samples/dotnet_core_module_sample/README.md

```diff
@@ -27,8 +28,7 @@ How does the data flow through the Gateway
 ------------------------------------------
 You can find the diagram for Receiving a message and publishing a message on this flow chart:
 
-![](../../bindings/dotnetcore/devdoc/images/flow_chart.png)
-
+![](../../bindings/dotnetcore/devdoc/images/flow_chart_native_gateway.png)
```

Comment:
This link got removed from this doc on latest version, per Code review comment on last pr. Please, remove.